products.json

```diff
@@ -54,5 +54,19 @@
         "artifactory-user": "ARTIFACTORY_USER",
         "artifactory-token": "ARTIFACTORY_TOKEN",
         "tarball-regex": "opensearch-*.tar.gz"
+    },
+    {
+        "name": "opensearch-plugin-prometheus-exporter",
+        "lp-building-project": "soss",
+        "lp-releasing-project": "opensearch-plugin-prometheus-exporter",
+        "lp-building-repo": "~data-platform/soss/+source/prometheus-exporter-plugin-for-opensearch/+git/prometheus-exporter-plugin-for-opensearch/",
```

Comment:
I'm not sure if this is the right thing here. See:

https://code.launchpad.net/~data-platform/soss/+source/prometheus-exporter-plugin-for-opensearch/+git/prometheus-exporter-plugin-for-opensearch/

where the recommended Git origin is directly `git.launchpad.net/soss/+source/prometheus-exporter-plugin-for-opensearch`. No `~data-platform`. This is the "anomalie" that we see across our repos in SOSS: https://code.launchpad.net/~data-platform/+git

True that `opensearch-build` is also listed there as `lp:soss/+source/opensearch-build` yet it's registered here (above line 48) as the URL behind indicates: `~data-platform/soss/+source/opensearch-build/+git/opensearch-build/`

What do you think...?

Reply:
Thank U very much.

It lead to success: https://launchpad.net/opensearch-plugin-prometheus-exporter-releases